requirements.txt

```diff
@@ -1,17 +1,18 @@
-llama-index==0.11.4
-llama-index-indices-managed-vectara==0.2.1
+llama-index==0.11.9
```

Comment:
If you want the bugfix to be included (with managed-vectara version 0.2.2, llama-index must be 0.11.10